### PR TITLE
Add Use different account option in security/recovery code page

### LINKED ISSUE #6634: Add `Use a different account` to TOTP signin and Recovery code page
We should update these login pages to have these links. Clicking the link should return to the `/signin` page.

image

### PR DESCRIPTION
**Pull request by arshadkazmi42**
Fixes #6634 
-----------

- Add `Use different account option` in security page
- Add `Use different account option` in recovery page

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 741bfa495743033bb92dcb7bb48d8d403421b9d3
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: app/scripts/templates/sign_in_recovery_code.mustache [hunk 1]
```diff
@@ -28,8 +28,9 @@
 
     </form>
 
-    <div class="links">
-        <a id="use-backup-link" class="center" href="/signin_totp_code">{{#t}}Use security code{{/t}}</a>
+    <div class="links centered">
+      <a id="use-recovery-code-link" href="/signin_recovery_code">{{#t}}Use recovery code{{/t}}</a>
```
Comment (vbudhram) [inline app/scripts/templates/sign_in_recovery_code.mustache]: This should say `Use security code` and have `id="use-backup-link"`

File: app/scripts/templates/sign_in_totp_code.mustache [hunk 2]
```diff
@@ -28,8 +28,9 @@
 
     </form>
 
-    <div class="links">
-        <a id="use-recovery-code-link" class="center" href="/signin_recovery_code">{{#t}}Use recovery code{{/t}}</a>
+    <div class="links centered">
+        <a id="use-recovery-code-link" class="center" href="/signin_recovery_code">{{#t}}Use security code{{/t}}</a>
```
Comment (vbudhram) [inline app/scripts/templates/sign_in_totp_code.mustache]: This should be `Use recovery code`

File: app/scripts/templates/sign_in_recovery_code.mustache [hunk 3]
```diff
@@ -28,8 +28,9 @@
...
+      <a id="handle-login" href="/signin">{{#t}}Use a different account{{/t}}</a>
```
Comment (vbudhram) [inline app/scripts/templates/sign_in_recovery_code.mustache]: I don't think we need `id="handle-login"` on the links.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b60eb9e13c28ded56f56d0eb5198d18483d4e992
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 495fae74b1bdfde41fae70c263bff6415dd8a51d
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by arshadkazmi42** [comment]
@vbudhram Why is travis check failing? Have I missed something?

**Comment by vbudhram** [comment]
@arshadkazmi42 Mind rebasing from master branch? That might contain some of the fixes.

File: app/scripts/templates/sign_in_recovery_code.mustache [hunk 4]
```diff
@@ -28,8 +28,9 @@
 
     </form>
 
-    <div class="links">
-        <a id="use-backup-link" class="center" href="/signin_totp_code">{{#t}}Use security code{{/t}}</a>
+    <div class="links centered">
+      <a id="use-backup-link" href="/signin_recovery_code">{{#t}}Use security code{{/t}}</a>
```
Comment (vbudhram) [inline app/scripts/templates/sign_in_recovery_code.mustache]: This should be pointing to `href="/signin_totp_code"`

File: app/scripts/templates/sign_in_totp_code.mustache [hunk 5]
```diff
@@ -28,8 +28,9 @@
 
     </form>
 
-    <div class="links">
-        <a id="use-recovery-code-link" class="center" href="/signin_recovery_code">{{#t}}Use recovery code{{/t}}</a>
+    <div class="links centered">
+      <a id="use-recovery-code-link" class="center" href="/signin_recovery_code">{{#t}}Use recovery code{{/t}}</a>
```
Comment (vbudhram) [inline app/scripts/templates/sign_in_totp_code.mustache]: The `class="center"` isn't needed because `centered` specified in the div above.

**Comment by vbudhram** [comment]
@arshadkazmi42 This has been merged in https://github.com/mozilla/fxa-content-server/pull/6700. Thank you!

**Pull request closed without merging**